internal/api/integrations/k8s/k8s.go

```diff
@@ -77,9 +76,27 @@ func (k *K8sClient) ListAll(ctx context.Context, namespace string) ([]models.Rel
 					Repository: r.Spec.Chart.Repository,
 					Version:    r.Spec.Chart.Revision,
 				},
+				Docker: dockerArtifacts(r),
 			},
 		})
 	}
 
 	return releases, nil
 }
+
+func dockerArtifacts(hr shipitv1beta1.HelmRelease) []models.DockerArtifact {
+	var artifacts []models.DockerArtifact
+
+	// find all "image" sections in the release's values, transforming each
+	// one into a docker artifact
+	unstructured.FindAll(hr.HelmValues(), "image", func(x interface{}) {
```

Comment:
Just being the devil's advocate, but what if there's an `image` value somewhere that isn't a docker image? Should we have a guard here to check if `repository` and `tag` are present?